apps/prairielearn/src/lib/server-jobs.ts

```diff
@@ -95,22 +95,30 @@ class ServerJobImpl implements ServerJob, ServerJobExecutor {
     this.addToOutput(chalk.blueBright(`Command: ${file} ${args.join(' ')}\n`));
     this.addToOutput(chalk.blueBright(`Working directory: ${options.cwd}\n`));
 
+    const start = Date.now();
```

Comment:
Nit: are we supposed to use [`performance.now()`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now) for timing?

Reply:
We probably should, yes!

Reply:
Done in 7640839.